src/test/java/teammates/test/cases/logic/TeamEvalResultTest.java

```diff
@@ -24,256 +24,268 @@ public static void setUp() {
+                {   0,   0,   0, NSU }, 
+                {   0,   0,   0, NSU },
+                { NSB, NSB, NSB, NSB },
+                {   0,   0, NSU, NSU }
+        };
+
+        int[][] expected6 = {
+                {   0,   0,   0, NSU }, 
+                {   0,   0,   0, NSU },
+                { NSB, NSB, NSB, NSB },
+                {   0,   0, NSU, NSU },
+
+                { NA,  0,  0, NA }, 
+                {  0, NA,  0, NA },
+                { NA, NA, NA, NA },
+                {  0,  0, NA, NA },
+
+                { 0, 0, 0, NA },
+
+                { 0, 0, 0, NA }, 
+                { 0, 0, 0, NA },
+                { 0, 0, 0, NA },
+                { 0, 0, 0, NA }
+        };
         verifyCalculatePoints(input6, expected6);
         
         //only one person submitted
-        int[][] input7 = 
-            {{  25,  25,  75 }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB }};
-        
-        int[][] expected7 = 
-            {{  60,  60, 180 }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB },
-             
-             {  NA,  50, 150 }, 
-             {  NA,  NA,  NA },
-             {  NA,  NA,  NA },
-             
-             {  NA,  50, 150 },
-             
-             {  NA,  25,  75 }, 
-             {  NA,  50,  150 },
-             {  NA,  50,  150 }};
+        int[][] input7 = {
+                {  25,  25,  75 }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB }
+        };
+
+        int[][] expected7 = {
+                {  60,  60, 180 }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB },
+
+                {  NA,  50, 150 }, 
+                {  NA,  NA,  NA },
+                {  NA,  NA,  NA },
+
+                {  NA,  50, 150 },
+
+                {  NA,  25,  75 }, 
+                {  NA,  50,  150 },
+                {  NA,  50,  150 }
+        };
 
         verifyCalculatePoints(input7, expected7);
-        
+
         //only one person, submitted for self only
-        int[][] input8 = 
-            {{  25, NSU, NSU }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB }};
-        
-        int[][] expected8 = 
-            {{ 100, NSU, NSU }, 
-             { NSB, NSB, NSB },
-             { NSB, NSB, NSB },
-             
-             {  NA,  NA,  NA }, 
-             {  NA,  NA,  NA },
-             {  NA,  NA,  NA },
-             
-             {  NA,  NA, NA },
-             
-             {  NA,  NA,  NA }, 
-             {  NA,  NA,  NA },
-             {  NA,  NA,  NA }};
+        int[][] input8 = {
+                {  25, NSU, NSU }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB }
+        };
+
+        int[][] expected8 = {
+                { 100, NSU, NSU }, 
+                { NSB, NSB, NSB },
+                { NSB, NSB, NSB },
+
+                {  NA,  NA,  NA }, 
+                {  NA,  NA,  NA },
+                {  NA,  NA,  NA },
```

Comment:
Perhaps the whitespace padding could be standardised to be based on either the longest character length of elements in the matrix, or on the alignment of the curly brace. Currently it seems to be mixed. The whitespace padding of `expected7` and before seems to be based on the character length, while the whitespace padding of `expected8` seems to be based on alignment of curly brace.